.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
 
 install:
   - npm install -g npm@5.5.1 yarn@1.3.2
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      sudo apt-get install --yes cvs libtool;
```

Comment:
How about using the more modern `apt` instead of `apt-get`? And the more common `-y` instead of `--yes`? And why is `libtool` explicitly required here?